packages/database/src/migrations/20210603023401-AddFijiCOVIDVaccinePercentageOfPopulationVaccinatedMapOverlays-modifies-data.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const getOverlayId = (doseNum, level) =>
+  `FJ_COVID_TRACKING_Dose_${doseNum}_${level}_Percentage_Vaccinated`;
+
+const createMapOverlay = (doseNum, aggregationEntityType, level, overlayLevelName) => {
+  const numeratorDataElementCode = doseNum === 1 ? 'COVIDVac4' : 'COVIDVac8';
+  const denominatorAggregationEntityType =
+    aggregationEntityType === 'sub_district' ? undefined : aggregationEntityType;
+  const denominatorAggregationType =
+    aggregationEntityType === 'sub_district' ? undefined : 'SUM_PER_ORG_GROUP';
```

Comment:
having tried to put undefined as value for these configs, if that doesn't throw errors that should be fine.